src/org/labkey/filetransfer/FileTransferModule.java

```diff
@@ -66,31 +64,23 @@ public boolean hasScripts()
         return false;
     }
 
+    public static void registerAdminConsoleLinks()
+    {
+        AdminConsole.addLink(AdminConsole.SettingsLinkType.Configuration, "File Transfer", FileTransferController.getComplianceSettingsURL());
```

Comment:
I believe you can pass another param to the end of the addLink() call to set the permission level needed to see this link. I believe this action is currently set to only be operations only (i.e. site-admin). In that case, does the page currently show an error for the ApplicationAdmin?